src/oceanBuilder.hpp

```diff
@@ -1,18 +1,14 @@
 #pragma once
-#include <iostream>
-#include <fstream>
+
 #include <string>
-#include <random>
-#include "json.hpp"
-#include "utilities.hpp"
-#include "mapping.hpp"
 
-using json = nlohmann::json;
+// project dependencies
+#include "json.hpp"
 
 class oceanBuilder {
 public:
     // Function to generate a random match based on the template
-    json randomOcean();
+    nlohmann::json randomOcean();
```

Comment:
since this is a header file, prefer not using `using` statements, which leak to anyone who includes the header file. Thus, I've removed `using json = nlohmann::json;` and included the namespace in the type name here.